src/__tests__/article_Tests/rating.test.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
```

Comment:
These tests should be in the articleTests folder, it was renamed from article_Tests to articleTests, you have two folders